### PR TITLE
Functional Tests : Migration TS (BO - 11)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : Migration TS (BO - 11)
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Relative to #30263
| Related PRs       | N/A
| How to test?      | Nightly is :green_circle: : https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/3713783062
| Possible impacts? | N/A

### CODE REVIEW

File: tests/UI/pages/BO/international/locations/index.js [hunk 1]
```diff
@@ -143,7 +143,7 @@ class Zones extends BOBasePage {
         break;
 
       case 'select':
-        await this.selectByVisibleText(page, this.zonesFilterColumnInput(filterBy), value ? 'Yes' : 'No');
+        await this.selectByVisibleText(page, this.zonesFilterColumnInput(filterBy), value == '1' ? 'Yes' : 'No');
```
Comment (Progi1984) [inline tests/UI/pages/BO/international/locations/index.js]:
```suggestion
        await this.selectByVisibleText(page, this.zonesFilterColumnInput(filterBy), value === '1' ? 'Yes' : 'No');
```

**Review by NeOMakinG — DISMISSED** [review]
TS seems to help you a lot in finding the wrong types! That's cool

**Review by boubkerbribri — DISMISSED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Review by SD1982 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]
🥒

**Comment by boubkerbribri** [comment]
Thanks @Progi1984

**Pull request merged**